src/sql/workbench/contrib/notebook/find/notebookFindModel.ts

```diff
@@ -406,7 +406,8 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 			if (oldDecorationIndex < oldDecorationsLen) {
 				// (1) get ourselves an old node
 				do {
-					node = this._decorations[oldDecorationsIds[oldDecorationIndex++]].node;
+					let decorationNode = this._decorations[oldDecorationsIds[oldDecorationIndex++]];
+					node = decorationNode ? decorationNode.node : undefined;
```

Comment:
You could replace this null check with an optional chaining operator.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining